scripts/HoveredMidiRelative/HoveredMidiRelativeExt.py

```
@@ -665,25 +820,29 @@ def onCustomOpen(self):
-				if par_in_group.owner == _par.owner and par_in_group.name == _par.name:
-					return
-
-		if self.lastCachedChange:#
-			if f'{_par.owner.path}:{_par.name}' == self.lastCachedChange[0] and _par.eval() == self.lastCachedChange[1]:
-				return
-
-		# update display with the new value
-		self.display_manager.update_parameter_display(_par)
+		try:
+			# check if _par is a cached parameter and if the value matches the cached value do nothing
+			if isinstance(_par, ParGroup):
```

Ruff (0.14.3)

`ParGroup` may be undefined, or defined from star imports

(F405)